romanNumeralsExercise/romanNumeralsExercise.js

```diff
@@ -0,0 +1,143 @@
+const inputNumber = prompt("Enter any digit between one and 3000: ");
+
+const seperateInputNumbers = [];
+
+let zeros;
+
+// loop through each value in input, round off each value to nearest 1000,100 or 10
+function numberRoundOff() {   
+    if (inputNumber.length == 4) {
+        zeros = '000';
+        for (let i = 0; i<inputNumber.length; i++) {
+        const numberA = inputNumber[i] + zeros ;
+        zeros = zeros.slice(0,-1);
+        seperateInputNumbers.push(Number(numberA));
+        }
+    } else if (inputNumber.length == 3) {
+        zeros = '00';
+        for (let i = 0; i<inputNumber.length; i++) {
+        const numberA = inputNumber[i] + zeros ;
+        zeros = zeros.slice(0,-1);
+        seperateInputNumbers.push(Number(numberA));
+        }
+    } else if (inputNumber.length == 2) {
+        zeros = '0';
+        for (let i = 0; i<inputNumber.length; i++) {
+        const numberA = inputNumber[i] + zeros ;
+        zeros = zeros.slice(0,-1);
+        seperateInputNumbers.push(Number(numberA));
+        }
+    } else if (inputNumber.length == 1) {
+        seperateInputNumbers.push(Number(inputNumber));
+    } else if ( inputNumber.length > 4) {
+        console.log("The number you have inserted is too large")
+    }
+    else {
+        console.log('You have not inserted a number');
+    }
+    return seperateInputNumbers;
+}
+numberRoundOff();
+console.log('Input numbers seperated and rounded off: ')
+console.log(seperateInputNumbers);
+
+
+let romanNumeralValue = '';
+let testValue;
+let baseNum;
+
+function romanNumerals() {
```

Comment:
[Here's](https://javascript.plainenglish.io/leetcode-12-integer-to-roman-javascript-21459e45a0ee) another solution worth reading through :)

And [here](https://leetcode.com/problems/integer-to-roman/discuss/?currentPage=1&orderBy=hot&query=&tag=javascript) are various submissions from other people on LeetCode you can browse through.